crux-core/src/crux/tx/polling.clj

```diff
@@ -21,7 +21,9 @@
                     (if-let [^Message last-message (reduce (fn [last-message ^Message m]
                                                              (case (get (.headers m) :crux.tx/sub-topic)
                                                                :docs
-                                                               (db/index-doc indexer (.key m) (.body m))
+                                                               ;; TODO we'll likely be able to make this more efficient
```

Comment:
This comment still valid?

Reply:
I think do - we only get one doc message at a time here (without a refactor) so we can't make full use of index-docs

Reply:
refactored :) although did mean changing Moberg slightly, don't know what the appetite for this would be...? anyway, build's green - if you're both happy, I'm ready to merge :smile: